visualizer/filters-content.css

```
@@ -102,4 +102,6 @@
   white-space: normal;
 }
 
-
+.filters-content .options #option-showoptimizationstatus .disabled {
```

```suggestion
.filters-content .options #option-showoptimizationstatus.disabled {
```
think this might be the issue: with the space, the CSS looks for an element with `class=disabled` _inside_ the showoptimizationstatus option element, but the code sets the class _on_ that option element.

---

@goto-bus-stop - That was part of the problem. Thanks!